src/mappers/provinces/province_mapper_importer.h

```diff
@@ -22,7 +23,7 @@ namespace mappers
 class ProvinceMapperImporter
 {
   public:
-   ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem);
+   ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem, const std::optional<vic3::World>& vic3_world);
```

Comment:
```suggestion
   ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem, const std::map<std::string, vic3::StateRegion>& state_regions_from_save);
```

You're only using the one bit from the source world, no need to pass the whole thing in.

Reply:
Can I still make it optional? In the test file, I don't have Vic3 states to include, and it's only used for logging.

Reply:
fix with [pass only StateRegion instead of the entire world](https://github.com/ParadoxGameConverters/Vic3ToHoI4/pull/632/commits/47a286889837d9214b1695c6424181cf51bfbe79) ?